### PR TITLE
Added AzureSQL

### LINKED ISSUE #23: MySQL
Deploy MySQL instance

### PR DESCRIPTION
**Pull request by zachChilders**
Adding SQL to our deployment.  It's pretty obviously not hardened.

Closes #23

### CODE REVIEW

File: tf/k8s.tf [hunk 1]
```diff
@@ -60,4 +60,13 @@ resource "azurerm_kubernetes_cluster" "k8s" {
...
-}
\ No newline at end of file
+}
+
+resource "azurerm_sql_server" "sql" {
+  name                         = "mics-pegasus"
+  resource_group_name          = "${azurerm_resource_group.rg.name}"
+  location                     = "West US"
+  version                      = "12.0"
+  administrator_login          = "4dm1n157r470r"
```
Comment (zachChilders) [inline tf/k8s.tf]: Lol, I think we have to use actual credentials.  I'm going to plumb these through the secrets pipeline.

File: tf/k8s.tf [hunk 2]
```diff
@@ -60,4 +60,36 @@ resource "azurerm_kubernetes_cluster" "k8s" {
...
+    geo_redundant_backup  = "Disabled"
+  }
+
+  administrator_login          = "${var.sql_user}"
+  administrator_login_password = "${var.sql_password}"
+  version                      = "5.7"
+  ssl_enforcement              = "Enabled"
+}
+
+resource "azurerm_mysql_database" "db" {
```
Comment (zachChilders) [inline tf/k8s.tf:89]: This creates a dummy db in the server.

**Comment by zachChilders** [comment]
Hardening options:
- Add firewall rule to only kubernetes
- Attach to VNET for segmentation

**Comment by lisnter** [comment]
Hey Zach.  I looked at this but I'm not sure what I'm looking for/at.  There's a conflict with Deploy-Infrastructure.ps1 - how do we resolve that?  Sorry.

**Review by v-Zer0 — APPROVED** [review]

**Comment by zachChilders** [comment]
@lisnter just wanted another pair of eyes to make sure I'm not doing anything blatantly stupid :)

It's also good to get everyone on the same page in the code base.

**Pull request merged**